src/operator/tensor/sample_op.cc

```diff
@@ -47,36 +51,86 @@ MXNET_OPERATOR_REGISTER_SAMPLE(random_normal, SampleNormalParam)
 .add_alias("_sample_normal")
 .describe(R"code(Draw random samples from a normal (Gaussian) distribution.
 
-Examples::
+.. note:: The existing alias ``normal`` is deprecated.
+
+Samples are distributed according to a normal distribution parametrized by loc (mean) and scale (standard deviation).
 
-  normal(loc=0, scale=1, shape=(2,2)) = [[ 1.89171135, -1.16881478],
-                                         [-1.23474145,  1.55807114]]
+Example::
+
+   random_normal(loc=0, scale=1, shape=(2,2)) = [[ 1.89171135, -1.16881478],
+                                                 [-1.23474145,  1.55807114]]
 )code" ADD_FILELINE)
 .set_attr<FCompute>("FCompute<cpu>", SampleNormal_<cpu>);
 
 MXNET_OPERATOR_REGISTER_SAMPLE(random_gamma, SampleGammaParam)
 .add_alias("_sample_gamma")
-.describe("Sample a gamma distribution")
+.describe(R"code(Draw random samples from a gamma distribution.
+
+Samples are distributed according to a gamma distribution parametrized by alpha (shape) and beta (scale).
```

Comment:
alpha -> *alpha* 
beta -> *beta*

So that we are following a consistent formatting for these distribution parameters.